app/pages/theme/style.css

```diff
@@ -16,6 +16,21 @@ input, button, textarea {
   border: 1px solid var(--ag-theme-primary);
 }
 
+select, select * {
+    background: var(--ag-theme-background);
+    font-size: inherit;
+    margin-bottom: 1em;
+    color: var(--ag-theme-text);
+    border: 0.05em solid var(--ag-theme-secondary);
```

Comment:
What do you think about having the border be the same as the `button` and `textarea`? I kinda like the current style there. I might just play around a bit after merging